File: Beecrowd/Beginner./1020.cpp
#include <iostream>
using namespace std;
int main () {
    int n;
    cin >> n;
    int year = n / 365;
    int month = (n % 365) / 30;
    int days = (n % 365) % 30 ;

    cout << year << " ano(s)\n" << month << " mes(es)\n" << days << " dia(s)" << endl;
    
    return 0;

}

File: Beecrowd/1. Beginner/1020/1020.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        int days = input.nextInt();

        int years = days/365;
        days %= 365;

        int months = days / 30;
        days %= 30;

        int day = days;

        System.out.println(years + " ano(s)");
        System.out.println(months + " mes(es)");
        System.out.println(day + " dia(s)");
        
        input.close();
    }
}